code/modules/multiz/zmimic/mimic_common.dm

```diff
@@ -9,11 +9,11 @@
 
 	return istype(T, /turf/space)
 
-/turf/update_icon()
+/turf/on_update_icon()
 	..()
 	if (above)
 		update_above()
 
-/atom/movable/update_icon()
+/atom/movable/on_update_icon()
 	..()
 	UPDATE_OO_IF_PRESENT
```

Comment:
This defeats the purpose: everything hard-overrides on_update_icon, so these updates will never fire except on atoms where they do nothing.

Reply:
Original `update_icon()`-calls restored.